### PR TITLE
Add MediaWiki guide

### PR DESCRIPTION
**Pull request by DougInAMug**
This is an initial draft for a MediaWiki installation. I see now the contributing guidelines suggest opening an issue first... sorry I didn't do that. (There is now #1759 :sweat_smile:)

I wrote it as I puzzled my way through, but then went through from the beginning again to check it all worked.

Advanced configuration, updates and backups not included.

### CODE REVIEW

**Review by nichtmax — CHANGES_REQUESTED** [review]
Thanks for the guide! Just a few things:

* Several links are broken in your preview, check out https://deploy-preview-1758--uberlab.netlify.app/guide_mediawiki/
* Please install directly into the DocumentRoot, not into https://<UberspaceUsername>.uber.space/mediawiki-1.41.1

**Comment by DougInAMug** [comment]
@nichtmax I corrected one broken link, but couldn't see any others! What am I missing? :see_no_evil: 

I don't understand your second point. The guide asked users to change into the document root, and install to there.

File: source/guide_mediawiki.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+Official, generalist installation manual for Mediawiki: `https://www.mediawiki.org/wiki/Manual:Installing_MediaWiki <https://www.mediawiki.org/wiki/Manual:Installing_MediaWiki>`_
+
+Change into your :manual:`document root <web-documentroot>`, download the latest MediaWiki release found at `https://www.mediawiki.org/wiki/Download <https://www.mediawiki.org/wiki/Download>`_ (here 1.41.1) and extract.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ cd html
+  [isabell@stardust html]$ wget https://releases.wikimedia.org/mediawiki/1.41/mediawiki-1.41.1.tar.gz
+  [isabell@stardust html]$ tar -xzvf mediawiki-*.tar.gz
```
Comment (nichtmax) [inline source/guide_mediawiki.rst:73]: Please add `--strip-components=1` to your tar command, then the installation will be in the document root without any sub folder like `/mediawiki-1.41.1`.

File: source/guide_mediawiki.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  [isabell@stardust html]$ rm mediawiki-*.tar.gz
+
+At this point, the wiki installer should be available at ``https://<UberspaceUsername>.uber.space/mediawiki-1.41.1/``, click *"complete the installation"* to start the graphical `configuration <https://www.mediawiki.org/wiki/Manual:Config_script>`_. Leave default values or change as desired. On the "Connect to database" page:
```
Comment (nichtmax) [inline source/guide_mediawiki.rst:76]: Please remove `mediawiki-1.41.1`.

File: source/guide_mediawiki.rst [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+* Database name: ``<UberspaceUsername>_mediawiki``
+* Database username: ``<UberspaceUsername>``
+* Database password: find out using ``my_print_defaults client``
+
+When the graphical installation completes, locally download the ``LocalSettings.php`` that should have been automatically generated and transfer it to ``~/html/mediawiki-1.41.1`` on your Uberspace. This can be achieved using one of the tools suggested on :manual:`SFTP <basics-SFTP>` with host ``sftp://<UberspaceUsername>.uber.space`` and your Uberspace credentials.
```
Comment (nichtmax) [inline source/guide_mediawiki.rst:82]: Please remove `mediawiki-1.41.1`.

File: source/guide_mediawiki.rst [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+After this, the installed wiki should be available under ``https://<UberspaceUsername>.uber.space/mediawiki-1.41.1``
```
Comment (nichtmax) [inline source/guide_mediawiki.rst:84]: Please remove `mediawiki-1.41.1`.

**Review by nichtmax — CHANGES_REQUESTED** [review]
I can't edit your commit directly, otherwise I would have made the small changes by myself ☺️ 
With installation into the document root we mean no `/mediawiki-1.41.1` in the URL.

**Comment by DougInAMug** [comment]
Hey @nichtmax I hope that's OK now!

**Pull request merged**